GVRf/Framework/src/org/gearvrf/GVRRenderData.java

```diff
@@ -261,7 +279,12 @@ public boolean isDone() {
             return mFutureMesh.isDone();
         }
     }
-
+    
+    public void addPass(GVRMaterial material, int cullFace) {
+    	NativeRenderData.addPass(getNative(), material.getNative(), cullFace);
+    }
+    
```

Comment:
Since cullFace only has 3 values here. Probably an enum here as parameter is more accurate.
Just FYI, if I were you, I may write an enum like
public static enum GVRCullFaceEnum{
      Back(0), Front(1), None(2);
      private final int value;
      private GVRCullFaceEnum(int value){
             this.value = value;
      }
      int getValue(){
            return value;
      }
}

public void addPass(GVRMaterial material, GVRCullFaceEnum cullFace){
        NativeRenderData.addPass(getNative(), material.getNative(), cullFace.getValue());
}

By this, you can restrict the developers that they can only use 0,1,2 to set cullFace.
Just FYI, not a big deal here.